src/runtime/GlobalObjectBuiltinString.cpp

```diff
@@ -1007,6 +1090,15 @@ void GlobalObject::installString(ExecutionState& state)
     m_stringPrototype->defineOwnPropertyThrowsException(state, ObjectPropertyName(strings->match),
                                                         ObjectPropertyDescriptor(new FunctionObject(state, NativeFunctionInfo(strings->match, builtinStringMatch, 1, nullptr, NativeFunctionInfo::Strict)), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::WritablePresent | ObjectPropertyDescriptor::ConfigurablePresent)));
 
+    FunctionObject* normalizeFunction = new FunctionObject(state, NativeFunctionInfo(strings->normalize, builtinStringNormalize, 1, nullptr, NativeFunctionInfo::Strict));
+    // The length property of the normalize method is 0.
+    normalizeFunction->setLength(1);
```

Comment:
Do you have special reason of setting length to 1?
I think we don't need FunctionObject::setLength and you can set length to 0 in ctor of NativeFunctionInfo

Reply:
It's my mistake. I set the length of normalize function to 0 in NativeFunctionInfo parameter.